Snippets/otf2ttf.py

```diff
@@ -1,13 +1,17 @@
 #!/usr/bin/env python
-from __future__ import print_function, division, absolute_import
```

Comment:
maybe just leave absolute_import

Reply:
What would it do here?

Reply:
on py2, would prevent from accidentally do a relative import without an explicit "." notation

Reply:
in the case of this snippet it probably doesn't matter, but I prefer to always add absolute_import in my lib modules

Reply:
Easier to just have all three there across the codebase, even if unused.